Podfile

```diff
@@ -11,5 +11,5 @@ target 'TeadsSampleApp' do
   pod 'GoogleMobileAdsMediationTeads'
   pod 'MoPub-Teads-Adapters'
   pod 'TeadsSASAdapter'
-  pod 'mopub-ios-sdk', '<= 5.15.0'
+  pod 'mopub-ios-sdk'
```

Comment:
you can remove all line here: dependency declaration is owned by MoPub-Teads-Adapters